src/cloudinary.coffee

```diff
@@ -141,16 +141,29 @@ class Cloudinary
     url
 
   ###*
-   * Generate an resource URL.
+   * Generates a URL for any asset in your Media library.
    * @function Cloudinary#url
-   * @param {string} publicId - the public ID of the resource
-   * @param {Object} [options] - options for the tag and transformations, possible values include all {@link Transformation} parameters
-   *                          and {@link Configuration} parameters
-   * @param {string} [options.type='upload'] - the classification of the resource
-   * @param {Object} [options.resource_type='image'] - the type of the resource
-   * @return {string} The resource URL
+   * @param {string} publicId - The public ID of the media asset.
+   * @param {Object} [options] - The {@link Transformation} parameters to include in the URL.
+   * @param {string} [options.type='upload'] - The asset's storage type. <p>Possible values:
+   * - `upload`
+   * - `private`
+   * - `authenticated` 
+   * - `sprite` 
+   * - `fetch`
+   * - A social media fetch type (Ex: `facebook`, `twitter`) 
+   * - A video thumbnail fetch type (Ex: `youtube`, `vimeo`).  
+   *For details on all fetch types, see <a href="https://cloudinary.com/documentation/image_transformations#fetching_images_from_remote_locations" target="_new">Fetch types</a>. 
```

Comment:
Is this and 2 lines above describe additional values of `upload` or of `fetch`.

Also, missing space before the "For"